src/scss/main.scss

```diff
@@ -499,6 +594,18 @@ $curve-springy: cubic-bezier(0.43, 0.48, 0.62, 1.07);
   display: block;
 }
 
+[data-braintree-id="methods-container"] {
```

Comment:
Would slightly prefer having a `braintree-methods-container` class or something along those lines here. We've loosely been trying to keep visual styles associated with class names and the hiding and showing functionality associated with the braintree id data attributes.

We could update the `braintree-options-list` to be `braintree-options-container` to keep this structure for both views, but we can do that as a different PR to keep it in scope.